string id
float32 pos
float32 raw_pos
float32 vel
float32 raw_vel
float32 kf_pos
float32 kf_vel
float32 kfb_pos
float32 kfb_vel
float32 kfb_bias
float32[] reserves

